source/template.yaml

```diff
@@ -189,6 +300,7 @@ Resources:
           - !GetAtt ProofAddressMockFunction.Arn
           - !GetAtt ProofResolutionFunction.Arn
           - !GetAtt ProofResolutionMockFunction.Arn
+          - !GetAtt ProofDocumentFunction.Arn
```

Comment:
one more reason to back out these changes --- this doesn't include the `ProofDocumentMockFunction`, luckily that will be autogenned now